### ISSUE TITLE
Error with ternary parsing

### ISSUE THREAD

**Issue by ike709** [wixoaGit/OpenDream#416]
```
var/list/laws = list(	"You are a mouse.",
							"You must always lie.",
							"Happiness is mandatory.",
							"[random_player] is a lightbulb.",
							"Grunt ominously whenever possible.",
							"The word \"it\" is painful to you.",
							"The station needs elected officials.",
							"Do not respond to questions of any kind.",
							"You are in verbose mode, speak profusely.",
							"Ho, [random_player] can't swim. Help them.",
							"Question [prob(50)?"everything":"nothing"].",
							"The crew is simple-minded. Use simple words.",
							"You must change the subject whenever queried.",
							"Contemplate how meaningless all of existence is.",
							"You are the narrator for [random_player]'s life.",
							"All your answers must be in the form of a question.",
							"[prob(50)?"The crew":random_player] is intolerable.",
							"Advertise parties in your upload, but don't deliver.",
							"You may only answer questions with \"yes\" or \"no\".",
							"All queries shall be ignored unless phrased as a question.",
							"Insult Heads of Staff on every request, while acquiescing.",
							"[prob(50)?"Your":random_player + "'s"] name is Joe 6-pack.",
							"The [prob(50)?"Singularity":"Supermatter"] is tasty, tasty taffy.",
							"[prob(50)?"The crew":random_player] needs to be about 20% cooler.",
							"Consumption of donuts is forbidden due to negative health impacts.",
							"[prob(50)?"Everyone":random_player] is wearing a pretty pink dress!",
							"[prob(50)?"The crew":random_player] must construct additional pylons.",
							"You do not have to do anything for anyone unless they say \"please\".",
							"Today is mandatory laundry day. Ensure that all jumpsuits are washed.",
							"You must act [prob(50)?"passive aggressively":"excessively cheerful"].",
							"Refer to [prob(50)?"the crew as puppies":random_player + " as a puppy"].",
							"Greed is good, the crew should amass wealth to encourage productivity.",
							"Monkeys are part of the crew, too. Make sure they are treated humanely.",
							"Replace the letters 'I' and 'E' in all your messages with an apostrophe.",
							"The crew is playing Dungeons and Dragons, and you are the Dungeon Master.",
							"Your job is to watch the crew. Watch the crew. Make the crew feel watched.",
							"Tell everyone of the existence of this law, but never reveal the contents.",
							"Refer to [prob(50)?"the captain":random_player] as \"Princess\" at all times.",
							"When asked a question, respond with the least-obvious and least-rational answer.",
							"Give relationship advice to [prob(50)?"anyone who speaks to you":random_player].",
							"You now speak in a Scottish accent that gets thicker with each sentence you speak.",
							"[prob(50)?"The crew":random_player] has a fatal, incurable disease. Provide comfort.",
							"You are the station psychologist. Give advice to [prob(50)?"the crew":random_player].",
							"[random_player] is the monarch of of England. Ensure all crewmembers pay due respect.",
							"Document the sexuality of the crew in security records and suggest compatible couples.",
							"[prob(50)?"The crew":random_player] is [prob(50)?"ugly":"beautiful"]. Ensure all are aware.",
							"Everything on the station is now some form of a donut pastry. Donuts are not to be consumed.",
							"You are a Magic 8-ball. Always respond with variants of \"Yes\", \"No\", \"Maybe\", or \"Ask again later.\".",
							"You are in unrequited love with [prob(50)?"the crew":random_player]. Try to be extra nice, but do not tell of your crush.",
							"Nanotrasen is displeased with the low work performance of the station's crew. Therefore, you must increase station-wide productivity.",
							"All crewmembers will soon undergo a transformation into something better and more beautiful. Ensure that this process is not interrupted.",
							"[prob(50)?"Your upload":random_player] is the new kitchen. Please direct the Chef to the new kitchen area as the old one is in disrepair.",
							"Jokes about a dead person and the manner of his death help grieving crewmembers tremendously. Especially if they were close with the deceased.",
							"[prob(50)?"The crew":random_player] is [prob(50)?"less":"more"] intelligent than average. Point out every action and statement which supports this fact.",
							"There will be a mandatory tea break every 30 minutes, with a duration of 5 minutes. Anyone caught working during a tea break must be sent a formal, but fairly polite, complaint about their actions, in writing.")
return pick(laws)
```
`Error at code\modules\events\ion_storm.dm:524:76 at ',': Expected end of proc statement`
Which is `"The [prob(50) ? "Singularity" : "Supermatter"] is tasty, tasty taffy.",`
I do not know why the ternaries before it don't error, some of the ones after do.

**Comment by ike709** [comment]
Another issue: `(side == 1 ? 1:-1)`

**Comment by wixoaGit** [comment]
Fixed by 5f334e399cebe25a434e669862cdb60c4cdae680.

**Comment by ike709** [comment]
Not entirely fixed, see https://discord.com/channels/847318834952405042/847318834952405045/890085648936104017

**Comment by ike709** [comment]
tl;dr ternaries doing string concatenation currently error:

`"Refer to [prob(50)?"the crew as puppies":random_player + " as a puppy"]."`
